pkg/server/container_start.go

```diff
@@ -300,6 +367,20 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount) {
 		// TODO(random-liu): [P1] Apply selinux label
 		g.AddBindMount(src, dst, options)
 	}
+	if privileged {
```

Comment:
I mean:
```
if !privileged {
  return
}
// or else
/*
clearReadyOnly...
*/
```

This reduces indent of the "clear read only code", which is a preferred coding style.